type_templates/vrm.js

```diff
@@ -93,25 +180,53 @@ export default e => {
   const _cloneVrm = async () => {
     const vrm = await parseVrm(arrayBuffer, srcUrl);
     vrm.cloneVrm = _cloneVrm;
+    vrm.toonShaderify = _toonShaderify;
     return vrm;
   };
 
   const _prepVrm = (vrm) => {
-    //vrm.visible = false; //will need later
+    vrm.visible = false;
     app.add(vrm);
     vrm.updateMatrixWorld();
     _addAnisotropy(vrm, 16);
   }
 
+  app.getActive = (_app = false) => {

```

Comment:
This is not a good function.

First of all it it's a getter that takes a parameter, which is bad.
Second, that parameter is not even consistent with the parameter of `setActive`.
Third, `getActive` sounds like a boolean. Is it active? But that is not what is returned here.
Fourth, the method is polymorphic and will return many different types.

Can we delete this?

Reply:
getActive is going to return the mesh that's corresponded with the current quality setting.  this only has support for base(high quality) and toon(ultra quality), but there will also be sprite and crunched.  I can rename for clarity, and remove the _app parameter so that only only 1 type is returned, and have the developer simply access .scene on their own.